tests/conftest.py

```diff
@@ -59,11 +62,21 @@ def db():
 
             conn.commit()
 
+            get_ipython().magic('load_ext sql')
+            get_ipython().magic(f'sql {created_database_url}')
+            # Set autopandas, because we think most users will
+            get_ipython().magic('config SqlMagic.autopandas=True')
+
             yield cur
         finally:
             cur.close()
             conn.close()
     finally:
+        for ipython_sql_connection in sql.connection.Connection.connections.values():
```

Comment:
Can you add a comment about why this is necessary? Or a link to docs, if that's what motivated these lines.